packages/block-editor/src/components/block-list/style.scss

```diff
@@ -554,10 +556,6 @@
 			margin-left: - $block-toolbar-height - $border-width;
 		}
 
-		.block-editor-block-contextual-toolbar[data-align="full"],
-		.block-editor-block-list__breadcrumb[data-align="full"] {
-			margin-left: 0;
```

Comment:
Why is this removed? This ensures that the movers are visible for "full" alignment.